visual-recognition/src/test/java/com/ibm/watson/visual_recognition/v4/model/CreateCollectionOptionsTest.java

```diff
@@ -51,9 +51,13 @@ public void testCreateCollectionOptions() throws Throwable {
     assertEquals(trainingStatusModel.objects(), objectTrainingStatusModel);
 
     CreateCollectionOptions createCollectionOptionsModel =
-        new CreateCollectionOptions.Builder().name("testString").description("testString").build();
```

Comment:
This file should have no changes

Reply:
Getting rid of trainingStatus there, and the Java-format script fixed its format too